main.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
files beginnen vaak lowercase. Je main runnable file -> is handig om deze main te noemen.